web/src/settings_org.ts

```diff
@@ -120,6 +120,31 @@ export function enable_or_disable_group_permission_settings(): void {
         return;
     }
 
+    if (current_user.is_admin) {
+        const $permission_pill_container_elements = $("#organization-permissions").find(
+            ".pill-container",
+        );
+        $permission_pill_container_elements.find(".input").prop("contenteditable", true);
+        $permission_pill_container_elements
+            .closest(".input-group")
+            .removeClass("group_setting_disabled");
+        settings_components.enable_opening_typeahead_on_clicking_label(
+            $("#organization-permissions"),
```

Comment:
OK. We'll want to refactor this later to use some sort of cleaner data-based check for whether the current user can modify the setting rather than all this hardcoding, but this is I guess OK for now.